### PR TITLE
fix(content-sidebar): Fix timing issue for additional tab loading state

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/additional-tabs/__tests__/AdditionalTabs-test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import AdditionalTabsLoading from '../AdditionalTabsLoading';
 describe('elements/content-sidebar/AdditionalTabs', () => {
     const getWrapper = props => shallow(<AdditionalTabs {...props} />);
 
-    test('should render the correct number of tabs and the loading state', () => {
+    it('should render the correct number of tabs and the loading state', () => {
```
Comment (ConradJChan) [inline src/elements/content-sidebar/additional-tabs/__tests__/AdditionalTabs-test.js]: should these be all `test`?

Comment (jstoffan) [inline src/elements/content-sidebar/additional-tabs/__tests__/AdditionalTabs-test.js]: Jest convention is to use `test`.

File: src/elements/content-sidebar/additional-tabs/AdditionalTabs.js [hunk 2]
```diff
@@ -27,40 +27,22 @@ class AdditionalTabs extends PureComponent<Props, State> {
...
-    /**
-     * Handles individual icon image load
-     *
-     * @return {void}
-     */
-    onImageLoad = () => {
-        const { tabs } = this.props;
+        let numTabs = tabs.length;
+        const hasMoreTab = tabs.find(tab => tab.id < 0 && !tab.iconUrl);
+        numTabs -= hasMoreTab ? 1 : 0;
```
Comment (jstoffan) [inline src/elements/content-sidebar/additional-tabs/AdditionalTabs.js]: General convention is to put `let` after `const`, which would also remove the need for reassignment.

**Pull request merged**